labelCloud/view/gui.py

```diff
@@ -88,6 +87,12 @@ def set_keep_perspective(state: bool) -> None:
         background: rgb(0, 0, 255);
         background: url("{icons_dir}/cube-outline_white.svg") center left no-repeat, #0000ff;
     }}
+
+    QComboBox#current_class_dropdown::item::selected{{
```

Comment:
Interesting idea with the bounding box icon.

Yet I would prefer a simple white font for the selected label.

Also is it possible to either only have the bounding box icon (and not the tick) or simply use background color an maybe make the previous one with a light gray background or similar?

- [ ] consider adapting

Reply:
Agree that it should be simpler. I changed it so no box icon and the background is greyed and the text color is white when hovering over.
<img width="283" alt="image" src="https://user-images.githubusercontent.com/60384727/189543727-0974d417-3458-4d1e-a396-ead891fc3e13.png">

Regarding the suggestion to style the previous active item, I'm not very familiar with the Qt stylesheet. I tried to google it https://doc.qt.io/qt-6/stylesheet-examples.html#customizing-qcombobox but I didn't find the option to style the previous active item. Please let me know if you know a way.

Reply:
What about the following?

```
    QComboBox#current_class_dropdown::item:checked
    {{
        color: gray;
    }}


    QComboBox#current_class_dropdown::item:selected {{
        color: #FFFFFF;
    }}

    QComboBox#current_class_dropdown{{
        selection-background-color: #0000FF;
    }}
```

Reply:
this looks great 👍 - included in another PR here [de5bced](https://github.com/ch-sa/labelCloud/pull/104/commits/de5bced85031404419d91974baab62e4152abfc1)